src/constructs/rds/instance.test.ts

```diff
@@ -0,0 +1,135 @@
+import "@aws-cdk/assert/jest";
+import { SynthUtils } from "@aws-cdk/assert/lib/synth-utils";
+import { Vpc } from "@aws-cdk/aws-ec2";
+import { DatabaseInstanceEngine, ParameterGroup, PostgresEngineVersion } from "@aws-cdk/aws-rds";
+import { App, Stack } from "@aws-cdk/core";
+import type { SynthedStack } from "../../../test/utils/synthed-stack";
+import { GuStack } from "../core";
+import { GuDatabaseInstance } from "./instance";
+
+describe("The GuDatabaseInstance class", () => {
+  const vpc = Vpc.fromVpcAttributes(new Stack(), "VPC", {
+    vpcId: "test",
+    availabilityZones: [""],
+    publicSubnetIds: [""],
+    privateSubnetIds: [""],
+  });
+
+  it("removes the db prefix from the input instance type", () => {
```

Comment:
Not sure I understand this test; `GuDatabaseInstance` gets instantiated with `instanceType: db.t3.small` and the `DBInstanceClass` is the same value? Should `DBInstanceClass` be `t3.small` in accordance to the test description?

Reply:
The logic being tested here removes the `db.` prefix if it exists on the instanceType value. This is because there's some logic in the CDK which adds the prefix regardless of whether it exists already. Without this logic the `DBInstanceClass` ends up being `db.db.t3.small` given the inputs of this test. I'll update the test name and/or add some comments as this isn't particularly clear.

It might be that we want to enforce the lack of a `db.` prefix in some other way so maybe let's have a think if there's a better way of doing it.